grpc/rpc/bmc.go

```diff
@@ -71,6 +71,38 @@ func (b *BmcService) Reset(ctx context.Context, in *v1.ResetRequest) (*v1.ResetR
 	return &v1.ResetResponse{TaskId: taskID}, nil
 }
 
+// DeactivateSOL deactivates any active SOL session on the BMC.
+func (b *BmcService) DeactivateSOL(ctx context.Context, in *v1.DeactivateSOLRequest) (*v1.DeactivateSOLResponse, error) {
+	l := logging.ExtractLogr(ctx)
+	taskID := xid.New().String()
+	l = l.WithValues("taskID", taskID)
+	l.Info(
+		"start DeactivateSOL request",
+		"username", in.Authn.GetDirectAuthn().GetUsername(),
+		"vendor", in.Vendor.GetName(),
+	)
+
+	execFunc := func(s chan string) (string, error) {
+		t, err := bmc.NewBMCResetter( // FIXME: BMCResetter?
```

Comment:
What's the FIXME about? Presumeably copy-n-paste from somewhere. 

```suggestion
		t, err := bmc.NewBMCResetter(
```

Reply:
Yeah, no idea at this point what that might have been about; removed...